pkg/scheduler/schedule_one.go

```diff
@@ -537,6 +537,9 @@ func (sched *Scheduler) findNodesThatPassFilters(
 	nodes []*framework.NodeInfo) ([]*v1.Node, error) {
 	numAllNodes := len(nodes)
 	numNodesToFind := sched.numFeasibleNodesToFind(fwk.PercentageOfNodesToScore(), int32(numAllNodes))
+	if !fwk.HasScorePlugins() {
```

Comment:
can you also check that there aren't any score extenders?

Reply:
I added checking if extender has `prioritizeVerb` configured - PTAL and let me know if that's alright.